blocks/timeline/index.js

```diff
@@ -234,28 +175,26 @@ registerBlockType( 'rtgb/timeline', {
 
 					<div className="content-wrap">
 						<div className="content-inner">
-							<h3 className="timeline-title">
-								<span className="timelineTitle">
-									{ timelineTitle }
-								</span>
+							<h4 className="timeline-title">
+								<span className="timelineTitle">{ title }</span>
+
 								{
 									( 'minor' === releaseType ) ? <span className="minor-release">{ __( '(Minor Release)' ) }</span> : ''
 								}
-							</h3>
-							<p className="timeline-description" title="Contributors">
-								<span className="dashicons dashicons-groups" />
+							</h4>
+							<p className="timeline-description">
 								<span className="contributor">
-									{ timelineContent }
+									{ content }
 								</span>
 							</p>
 							<div className="postlinks">
-								{ ( newslink || bloglink ) ? <strong>Links: </strong> : null }
+								{ ( newsLink || blogLink ) ? <strong>Links: </strong> : null }
 
 								{
-									newslink ? <a href={ newslink } className="timeline-news-link">{ __( 'News' ) }</a> : ''
+									newsLink ? <a href={ newsLink } className="timeline-news-link">{ __( 'News' ) }</a> : ''
```

Comment:
Can also write it like this
```
!! newsLink && <a href={ newsLink } className="timeline-news-link">{ __( 'News' ) }</a>
```
Other conditions can also be modified like this.

Reply:
Done